svnwiki-mode.el

```diff
@@ -330,4 +333,5 @@ by default.  It may introduce font-lock bugs and other errors."
     (svnwiki-initial-fontification)))
 
 (provide 'svnwiki-mode)
+
```

Comment:
Why whitespace changes?

Reply:
A blank line here is the convention in `.el` files that ship with Emacs.